src/app.py

```diff
@@ -48,8 +48,14 @@ def __init__(self, host, bucket, scope, collection, username, pw):
     def connect(self, **kwargs):
         # note: kwargs would be how one could pass in
         #       more info for client config
+
         conn_str = f"couchbase://{self.host}"
 
+        # Note: To connect with Couchbase Capella, use the following conn_str instead of the one above as it needs TLS
+        # Also ensure that the bucket, scopes & collections are created on Capella before running the application
+
+        # conn_str = f"couchbases://{self.host}?ssl=no_verify"
```

Comment:
What if I want to use SSL?

Reply:
I have updated the code to work with SSL using certificates.